specifyweb/frontend/js_src/lib/components/Formatters/Definitions.tsx

```diff
@@ -124,21 +114,29 @@ function ConditionalFormatter({
   readonly table: SpecifyTable;
 }): JSX.Element {
   const isReadOnly = React.useContext(ReadOnlyContext);
+
   const [isExpanded, _, __, handleToggle] = useBooleanState(false);
+
+  const handleDelete = (): void =>
+    handleChange(removeItem(formatter.definition.fields, index));
+
+  const expandedNoCondition = isExpanded || !hasCondition;
+
   return (
     <div
-      className={`flex ${
-        isExpanded || !hasCondition ? 'flex-col' : 'items-center'
-      } gap-2`}
+      className={`flex
+        ${isExpanded || !hasCondition ? 'flex-col' : 'items-center'}
+        ${isExpanded ? 'gap-2' : ''}
+      `}
       key={index}
     >
       {hasCondition && (
         <Label.Block>
           {isExpanded ? resourcesText.conditionFieldValue() : null}
-          <div className="flex items-center gap-2">
-            <div className="flex-1">
+          <div className="flex items-center gap-2 pr-2">
+            <div className="h-full flex-1">
               <Input.Text
-                className="h-full"
+                className="h-full pr-2"
```

Comment:
why does this have pr-2?
this seems like it would be better accomplished by gap

i.e you can just increase `gap-2` if you need to
or, you can remove the "-ml-2" from `<span className="-ml-2" />` (remember that the "-ml-2" is there to "eat the gap", though normally we have it as `<span className="flex-1 -ml-2" />`)